packages/@sanity/schema/package.json

```diff
@@ -77,13 +77,15 @@
     "@sanity/generate-help-url": "^3.0.0",
     "@sanity/types": "3.32.0",
     "arrify": "^1.0.1",
+    "groq-js": "1.5.0-canary.1",
```

Comment:
would it make sense to widen the range to `^1.5.0.canary.1`?

Reply:
Looks like it doesn't work, because it's not valid semver? 🤔 

```
 ERR_PNPM_SPEC_NOT_SUPPORTED_BY_ANY_RESOLVER  groq-js@^1.5.0.canary.1 isn't supported by any available resolver.
```

Reply:
my bad, should've been `^1.5.0-canary.1` (notice the dash)